workflow/rules/multiqc.smk

```diff
@@ -34,11 +34,12 @@ rule vcfStats:
 
 rule multiQC:
     input:
-        expand(["results/fastqc/{sample}_1_fastqc.html","results/fastqc/{sample}_2_fastqc.html"], sample=samples),
-        expand(["results/fastqc/{sample}_1_fastqc.zip", "results/fastqc/{sample}_2_fastqc.zip"], sample=samples),
-        expand("results/alignments/bamStats/{sample}.flagstat", sample=samples),
-        expand("results/coverage/{sample}.per-base.bed.gz", sample=samples),
-        expand("results/wholegenome/coverage/windowed/{sample}.regions.bed.gz", sample=samples),
+        expand(["results/fastqc/{sample}_1_fastqc.html","results/fastqc/{sample}_2_fastqc.html",
```

Comment:
this is cool, I haven't put expand statements over multiple lines in the input before :)